### DISCUSSION TITLE
Remove Default Styles

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#7060] [Help] [answered]
Is there some sort of config param allowing for removal of default styles?

**Comment by simonswiss** [comment, marked as answer]
Hey!

Tailwind's default styles are part of what's called [preflight](https://tailwindcss.com/docs/preflight), and you can disable these in your `tailwind.config.js` file by setting `preflight: false` in the `corePlugins` option:

```diff
module.exports = {
+  corePlugins: {
+    preflight: false,
+  }
}
```

Hope it helps!

**Reply by ghost** [reply]
Ahh, I'd skimmed and searched that page a couple of times but didn't make the connection that all default styles were in fact exactly for cross-browser compatibility like stated on that page. Wish I would've made that connection myself, thank you very much @simonswiss!
